bin/install-password-argon2

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+set -o errexit
+
+# If PHP < 7.2, exit
+if [[ $VERSION =~ ^master$ ]]; then
```

Comment:
@Jean85 Can you explain this condition? It doesn't make sense to me, master (aka nightly) > 7.2, it should build for master too. Thanks.

Reply:
I'm sorry, the original code wasn't mine (see #19) and it predated 7.2, so it may be incorrect right now...